src/components/Callout/style.css

```diff
@@ -0,0 +1,63 @@
+.Callout {
+  --rvr-callout-bg: var(--rvr-blue-lite-2);
+  --rvr-callout-border-color: var(--rvr-blue);
+  --rvr-callou-color: var(--rvr-gray-80);
+
+  border-radius: 2px;
+  box-sizing: border-box;
+  display: flex;
+  padding: 20px;
+  position: relative;
+  background-color: var(--rvr-callout-bg);
+  border-bottom-color: var(--rvr-callout-border-color);
+  border-bottom-style: solid;
+  border-bottom-width: 1px;
+  color: var(--rvr-callout-color);
+}
+
+.success {
+  --rvr-callout-bg: var(--rvr-green-lite-2);
```

Comment:
We might want to sanity check that this works in IE11. I know custom css props have given us troubles occasionally.

Reply:
That is actually a very good point. I'll take a look later this morning